### ISSUE TITLE
tool.py - from builtins import range

### ISSUE THREAD

**Issue by ronshnapp** [OpenPIV/openpiv-python#101]
in the tools.py, the import 
_from builtins import range_
resulted in an exception. not sure if this is needed. After commnting out, the tutorial-part1 notebook worked fine. 
(using python2.7. Installed through the anaconda)

**Comment by alexlib** [comment]
do you have the package `builtins` ? i think this is just our attempt to provide a consistent performance for both Python 2.7 and 3.5+ 

try first please 

``` conda install future ```

**Comment by ronshnapp** [comment]
yup, it works. So the future needs to be listed in conda as a dependency, so it is downloaded when installing through conda.

**Comment by alexlib** [comment]
@ronshnapp please how did you install? 

`conda install openpiv -c conda-forge` 

 ?

**Comment by ronshnapp** [comment]
Yes, copy - pasted from the  Readme in the front page.
